framework/PageDashboard/PageChartLegend.tsx

```diff
@@ -0,0 +1,103 @@
+import { Link } from 'react-router-dom';
+
+export function PageChartLegend(props: {
+  id: string;
+  legend: { label: string; count?: number; color: string; link?: string }[];
+  horizontal?: boolean;
+}) {
+  if (props.horizontal) {
+    return (
+      <div
+        style={{
```

Comment:
It would be good to utilize `styled-components` instead of the inline styling.

Reply:
Yes. I personally find inline styles to be easier to follow than having to follow a link to a style that is elsewhere in the code. Performance impacts in this case are small.